agent/engine/docker_task_engine.go

```diff
@@ -840,6 +840,12 @@ func (engine *DockerTaskEngine) pullContainer(task *apitask.Task, container *api
 
 	}
 
+	// Add the container to pulled container state
```

Comment:
could you add update the comment to note this is for cached images?

Reply:
+1

Reply:
The comment is updated to 
```
func (engine *DockerTaskEngine) pullContainer(task *apitask.Task, container *apicontainer.Container) dockerapi.DockerContainerMetadata {
	...
	if engine.imagePullRequired(engine.cfg.ImagePullBehavior, container, task.Arn) {...}

	// No pull image is required, the cached image will be used.
	// Add the container that uses the cached image to the pulled container state.
	dockerContainer := &apicontainer.DockerContainer{
		Container: container,
	}
	engine.state.AddPulledContainer(dockerContainer, task)
       ...
}
```

Thank you!